IntType.h

```diff
@@ -17,8 +20,18 @@ struct IntType
     IntType& multiply( int rhs );
     IntType& divide( int rhs );
 
+    IntType& pow(const IntType& i);
+    IntType& pow(const FloatType& f);
+    IntType& pow(const DoubleType& d);
+    IntType& pow(int i);
+
+    //Operators for Static Conversion
+    //Const objects can only call their const member functions
     operator int()  {return *intPtr;}
```

Comment:
You don't need a non-const version of this, since none of your functions take a non-const version.
- [ ] remove